### PR TITLE
Configurable API root for clients.

### PR DESCRIPTION
**Pull request by cmars**
Consolidate API root configuration & default, client construction patterns.

### CODE REVIEW

File: default.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package romulus contains service API clients related to metering & SLAs.
+package romulus
+
+var DefaultAPIRoot = "https://api.jujucharms.com/omnibus/v3"
```
Comment (alesstimec) [inline default.go]: godoc

Comment (cmars) [reply default.go]: Done!

**Comment by cmars** [comment]
$$merge$$

**Comment by cmars** [comment]
No landing bot, I'll just push the green button.

**Pull request merged**